# Array of potential obstacles in bubblescope image
CastObstacle[] obstacles
